### PR TITLE
Supported getting all items that the player has

### PR DESCRIPTION
**Pull request by kitlawes**
This pull request is intended to solve issue #3.

### CODE REVIEW

File: src/de/zabuza/sparkle/freewar/inventory/Inventory.java [hunk 1]
```diff
@@ -167,6 +168,35 @@ public boolean hasItem(final String item) {
...
 	 * (non-Javadoc)
 	 * 
+	 * @see
+	 * de.zabuza.sparkle.freewar.inventory.IInventory#getItems()
+	 */
+	@Override
+	public String[] getItems() {
+		openInventory();
+		
+		List<String> items = new ArrayList<String>();
```
Comment (ZabuzaW) [inline src/de/zabuza/sparkle/freewar/inventory/Inventory.java]: Missing final modifier

File: src/de/zabuza/sparkle/freewar/inventory/Inventory.java [hunk 2]
```diff
@@ -167,6 +168,35 @@ public boolean hasItem(final String item) {
...
+		
+		// Add unequipped items
+		List<WebElement> itemElements = m_Driver.findElements(By.cssSelector(CSSSelectors.ITEM_INVENTORY_ITEM_NAME));
```
Comment (ZabuzaW) [inline src/de/zabuza/sparkle/freewar/inventory/Inventory.java]: Missing final modifier

File: src/de/zabuza/sparkle/freewar/inventory/Inventory.java [hunk 3]
```diff
@@ -167,6 +168,35 @@ public boolean hasItem(final String item) {
...
+		for (WebElement itemElement : itemElements) {
```
Comment (ZabuzaW) [inline src/de/zabuza/sparkle/freewar/inventory/Inventory.java]: Missing final modifier

**Review by Zabuzard — APPROVED** [review]
Though the project has not used final modifiers a lot, it should do.
Method `getItems` should get some of them.

Besides that everything is fine. Great pull request!

**Pull request merged**